rerun_py/rerun_sdk/rerun/_log.py

```diff
@@ -218,6 +218,9 @@ def log_components(
         if None, use the global default from `rerun.strict_mode()`
 
     """
+    # Convert to a native recording
+    recording = RecordingStream.to_native(recording)
```

Comment:
This was just an error before on this API. Apparently we don't use stream-overrides in any of our other examples.